platformio.ini

```diff
@@ -73,8 +63,12 @@ lib_deps =
 ; https://community.platformio.org/t/gtest-not-working-on-pio-4-1/10465/7
 lib_compat_mode = off
 extra_scripts = platformio_sanitizers.py
-; Run clang-tidy only on native: it seems to get confused by Uno headers that
-; can only be parsed by gcc.
+
+; Run clang-tidy only on native: it seems to get confused by headers that can
+; only be parsed by gcc.
+;
+; TODO(jlebar): Is this still necessary now that we have dropped support for
+; Uno?  If so, will it still be necessary when we drop support for Nucleo?
```

Comment:
It will definitely be easier to use clangtidy when we don't have the Arduino SDK for it to parse. I'll try to remember to give it a shot when nucleo is gone.